ballerina/Module.md

```diff
@@ -1,18 +1,45 @@
-# Module Overview
+# Overview
 
-This module provides relation database support for the `bal persist` feature, which provides functionality to store and query data from a relational database conveniently through a data model instead of using SQL query language.
+This module provides relational database support for the `bal persist` feature, which provides functionality to store and query data from a relational database through a data model instead of writing SQL.
 
-Currently, this package supports MySQL, MSSQL, PostgreSQL as the relational database. However, we are also planning to add support for other relational databases such as Oracle.
+Currently, this package supports MySQL, MSSQL and PostgreSQL databases. However, we are also planning to add support for other relational databases such as Oracle.
 
-The MySQL, MSSQL, PostgreSQL data stores are relational database management systems that stores data in tables. None of these data stores is the default data store for the `bal persist` feature. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application. as follows,
+## How to use with `bal persist`
 
-```
-$ bal persist init --datastore mysql
-$ bal persist init --datastore mssql
-$ bal persist init --datastore postgresql
-```
+Since none of the relational databases is not the default datastore for `bal persist`, you need to explicitly specify the data store in your application in either of the following ways,
```

Comment:
Isn't this should be
```suggestion
Since none of the relational databases is set as the default datastore for `bal persist`, you need to explicitly specify the data store in your application in either of the following ways,
```
?

Reply:
In the `bal persist add` command, the default datastore is `inmemory` while in `bal persist generate` command, the datastore field is mandotory. Since `inmemory` is not a relational database, this statement is valid. Shall I go ahead and make the change?

Reply:
In that case, something like below would be more clear IMO. WDYT?

```
By default, `bal persist` utilizes the in-memory datastore. Therefore, if you intend to use a different datastore, you must explicitly specify it as follows:
```

Reply:
+1. Since this package is `persist.sql`, the need to explicitly specify the data store is implied. Hence, what do you think about this?

```
By default, `bal persist` utilizes the in-memory data store. Therefore, you must explicitly specify an SQL data store as follows:
```

Reply:
+1